scripts/dump_pairwise_model.py

```diff
@@ -148,5 +161,5 @@
 models = {}
 models["clusterer"] = cluster
 
-with open(join(CONFIG["main_data_dir"], "prod_model.pickle"), "wb") as f:
+with open(join(CONFIG["main_data_dir"], "prod_model_0.1.4.pickle"), "wb") as f:
```

Comment:
Need to include this new pickle file as part of the pull request?

Reply:
I renamed it to `prod_model.pickle`, put it in the `artifacts` folder (not part of github repo) and ran `tt publish-artifacts --artifacts-dir artifacts --s3-path s3://ai2-s2-research-public/paper_clustering_artifacts_v0_2.tar.gz`

Reply:
Is that sufficient? That's the pickle specified in the new variant in the config.